<launch>

<rosparam file="$(find dome)/config/all_cues.yaml" />


<!-- VISUAL -->
    <node name="domeVisualScene" pkg="dome" type="VisualScene">
        <param name="fullScreen" type="bool" value="false" />
        <param name="displayOnly" type="bool" value="false" />
        <param name="posX" type="int" value="500" />
        <param name="posY" type="int" value="100" />
        <param name="width" type="int" value="1000" />
        <param name="height" type="int" value="1000" />
        <param name="offset" type="double" value="-61.5" />
    </node>

</launch>
